InvenTree/build/views.py

```diff
@@ -29,7 +31,7 @@ def get_queryset(self):
 
     def get_context_data(self, **kwargs):
 
-        context = super(BuildIndex, self).get_context_data(**kwargs).copy()
```

Comment:
Are you sure we do not need that? This was the object stays coupled

Reply:
Do you mean the `super(BuildIndex, self)` call? It's now just `super()` (new style syntax) and the [MRO](https://realpython.com/python-super/#method-resolution-order) should sort it out.

Reply:
@SchrodingersGat I mean the .copy()

Reply:
I see. If you search for `super().get_context_data` it does not seem to be consistent.

The [django docs](https://docs.djangoproject.com/en/4.0/topics/class-based-views/generic-display/#adding-extra-context-1) seem to indicate that the `copy()` is not required.

Should we remove all `.copy()` calls?

Reply:
@SchrodingersGat I never use copy() with context as the object is created on each new request. But it should be consistent throughout the codebase.

Reply:
This can be a separate issue too. The rest of the PR seems fine and a good improvment. The change just jumped out.